src/builder.cc

```diff
@@ -72,6 +72,7 @@ Builder::Builder()
 
 Builder::~Builder()
 {
+    impl_->graphs.clear();
```

Comment:
This change may break the copy semantics of the Builder class. Once Builder object is copied and one of them is destructed, Graph object may removed even if another is alive.

```
Buidler a;
// construct graph

{
   Builder b = a;
}

// a doesn't hold graph any more
```